global.d.ts

```diff
@@ -39,10 +39,13 @@ export interface PlaywrightMatchers<R> {
   */
   toEqualText(value: string, options?: PageWaitForSelectorOptions): Promise<R>;
   /**
-  * Will ensure that the element is one the page in a given timeout (default 1 second).
+  * Will ensure that the element is on the page in a given timeout (default 1 second).
```

Comment:
```suggestion
  * Will ensure that the element is on the page.
```
we removed the default 1 second timeout a while ago if I'm correct. There are other places in the readme 

> By default it waits 1 second for the element which you can overwrite via the options.

3 times, would be great if you could remove them too.

Reply:
done